runtime/queries/nickel/highlights.scm

```diff
@@ -0,0 +1,74 @@
+(comment) @comment
+
+[
+  "if"
+  "then"
+  "else"
+  "forall"
+  "in"
+  "let"
+  "default"
+  "doc"
+] @keyword
+
+"fun" @keyword.function
+
+"import" @include
+
+[ "if" "then" "else" ] @conditional
+"switch" @conditional
```

Comment:
Can you update scopes like these to use the helix conventions? https://docs.helix-editor.com/themes.html#syntax-highlighting

Reply:
Will do!